CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 
-cmake_minimum_required(VERSION 2.8.3)
+# version 2.8.3 is deprecated so switch to 3.0
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Can you make it `3.0.2` so it's inline with SFML 2.5.1

Reply:
ok it's done